### DISCUSSION TITLE
cannot find method or associated constant `enum_type_name` in trait `sea_orm::sea_query::ValueType` when running migration

### DISCUSSION THREAD

**Discussion by dipthegeezer** [SeaQL/sea-orm#2437] [Q&A] [answered]
Hi 

First time user of sea orm and also fairly new to rust in general so not entirely sure where the problem is. Hopefully someone can help me. 

When I run 
```
sea-orm-cli migrate -u postgres://user:mypassword@localhost:5432/db up
```

I'm getting the following error:

```
Compiling sea-orm-migration v1.1.2
error[E0576]: cannot find method or associated constant `enum_type_name` in trait `sea_orm::sea_query::ValueType`
 --> /Users/xxx/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sea-orm-migration-1.1.2/src/seaql_migrations.rs:7:5
  |
7 |     #[sea_orm(primary_key, auto_increment = false)]
  |     ^ not found in `sea_orm::sea_query::ValueType`

error[E0576]: cannot find method or associated constant `enum_type_name` in trait `sea_orm::sea_query::ValueType`
 --> /Users/xxx/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sea-orm-migration-1.1.2/src/seaql_migrations.rs:9:5
  |
9 |     pub applied_at: i64,
  |     ^^^ not found in `sea_orm::sea_query::ValueType`
```

I've followed the tutorial to generate the migration files. I'm using postgres as my DB. Any help appreciated.  I'm using the latest version of rust 1.83.0

Thanks in advance.

**Comment by dipthegeezer** [comment, marked as answer]
Going to resolve this. Looks like my cargo.lock file got a bit corrupted or something. Not exactly sure but as soon as I rebuilt it the compilation errors went away. Resolving.

**Reply by behnam-oneschema** [reply]
I'm guessing if there's a dependency bump that needs to happen in `sea-orm`, maybe for `sea-query`?

**Comment by behnam-oneschema** [comment]
Fyi, I just filed a bug report for this, considering that my `Cargo.lock` is not corrupted: https://github.com/SeaQL/sea-orm/issues/2444
